aks/stratos-metrics-values.yaml

```diff
@@ -1,13 +1,7 @@
 env:
   DOPPLER_PORT: 443
-kubernetes:
-  apiEndpoint: MASTER_URL 
 prometheus:
   kubeStateMetrics:
     enabled: true
-nginx:
-  username: xxxx
-  password: xxxxx
 services:
   loadbalanced: true
```

Comment:
We need the metrics username/password here. In the latest metrics release the key is not `nginx` but `metrics` so it's `metrics.username` and `metrics.password`.

Reply:
I moved the metrics deploy to scfcharts.tf 3d9a3b36